### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by OleMortensen8**
This code of conduct is NOT Ready!!! look for your non-exposing email and add it so people can inform you about, if people act against the code.  be nice!  also need it in danish when it's changed DON'T merge just yet

### CODE REVIEW

**Comment by OleMortensen8** [comment]
have you checked my file to perfect it?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+
+Community Impact Guidelines were inspired by [Mozilla's code of conduct
+enforcement ladder](https://github.com/mozilla/diversity).
+
+[homepage]: https://www.contributor-covenant.org
+
```
Comment (subtostrikeman) [inline CODE_OF_CONDUCT.md:125]: add my mail here

**Review by subtostrikeman — APPROVED** [review]
its done

**Comment by subtostrikeman** [comment]
its done add it to main  my secret mail is in there

**Review by OleMortensen8 — COMMENTED** [review]
looks good!

**Review by subtostrikeman — APPROVED** [review]

**Review by subtostrikeman — APPROVED** [review]

**Review by subtostrikeman — APPROVED** [review]
loooks good! sorry for the wait as the owner of this repo!

**Comment by OleMortensen8** [comment]
I'll add some changes to try and allow the merge

**Comment by OleMortensen8** [comment]
**now i can't approve. some other than me need to approve the changes.**

**Review by subtostrikeman — APPROVED** [review]

**Pull request merged**